ansible_collections/arista/avd/roles/eos_designs/python_modules/flows/avdstructuredconfig.py

```diff
@@ -124,3 +124,102 @@ def _enable_sflow(self) -> bool:
                 return True
 
         return False
+
+    @cached_property
+    def _default_flow_tracker(self) -> dict:
+        return {
+            "name": self.shared_utils.default_flow_tracker_name,
+            "record_export": {"on_inactive_timeout": 70000, "on_interval": 300000},
+            "exporters": [{"name": "CV-TELEMETRY", "collector": {"host": "127.0.0.1"}, "local_interface": "Loopback0", "template_interval": 3600000}],
+        }
+
+    def resolve_flow_tracker_by_type(self, tracker_settings: dict) -> dict:
+        tracker = {
+            "name": tracker_settings["name"],
+            "record_export": tracker_settings.get("record_export"),
+            "exporters": tracker_settings.get("exporters"),
+        }
+        if self.shared_utils.flow_tracking_type == "sampled":
+            sampled_settings = get(tracker_settings, "sampled", {})
+            if (table_size := sampled_settings.get("table_size")) is not None:
+                tracker["table_size"] = table_size
+            if (mpls := get(sampled_settings, "record_export.mpls")) is not None:
+                if tracker["record_export"] is None:
+                    tracker["record_export"] = {}
+                tracker["record_export"]["mpls"] = mpls
```

Comment:
Would this make more sense ? I am not sure how `record_export` can be `None` - would you not get a KeyError instead?
```suggestion
            if (mpls := get(sampled_settings, "record_export.mpls")) is not None:
                tracker.setdefault("record_export", {})["mpls"] = mpls
```

Reply:
just did,  tracker["record_export"]["mpls"] = mpls